pkgs/top-level/python-packages.nix

```diff
@@ -21393,21 +21393,7 @@ in {
     };
   };
 
-  sqlmap = buildPythonPackage {
-    name = "sqlmap-1.0.11";
-
-    src = pkgs.fetchurl {
-      url = "mirror://pypi/s/sqlmap/sqlmap-1.0.11.tar.gz";
-      sha256 = "1x4amyjqnd9j5g2kp9nvg8pr5sqzbhr8gd0m6d671bshvgj568vr";
-    };
-
-    meta = with pkgs.stdenv.lib; {
-      homepage = "http://sqlmap.org";
-      license = licenses.gpl2;
-      description = "Automatic SQL injection and database takeover tool";
-      maintainers = with stdenv.lib.maintainers; [ bennofs ];
-    };
-  };
+  sqlmap = callPackage ../applications/misc/sqlmap { };
```

Comment:
I would not leave this here. It confuses users, it will be still available as `python3Packages.sqlmap`.
We have no `aliases.nix` for `python-packages.nix` unfortunally.